spec/services/proofing/lexis_nexis/ddp/proofing_spec.rb

```diff
@@ -55,24 +70,28 @@
         request =
           stub_request(
             :post,
-            verification_request.url,
+            proofing_verification_request.url,
           ).with(
-            body: verification_request.body,
-            headers: verification_request.headers,
+            body: proofing_verification_request.body,
+            headers: proofing_verification_request.headers,
           ).to_return(
             body: LexisNexisFixtures.ddp_success_response_json,
             status: 200,
           )
 
-        verification_request.send_request
+        proofing_verification_request.send_request
 
         expect(request).to have_been_requested.once
       end
     end
   end
 
-  subject do
-    described_class.new(LexisNexisFixtures.example_config.to_h)
+  subject(:proofing) do
+    described_class.new(LexisNexisFixtures.example_ddp_proofing_config.to_h)
+  end
+
+  subject(:authentication) do
+    described_class.new(LexisNexisFixtures.example_ddp_authentication_config.to_h)
   end
```

Comment:
Minor: I wouldn't expect two subjects in the same scope like this, I think I'd move the `subject` to be overridden in the context relevant for the account-creation policy tests.

e.g.

```rb
subject(:proofer) { described_class.new(LexisNexisFixtures.example_ddp_proofing_config.to_h) }

context 'when user is going through account creation' do
  subject(:proofer) { described_class.new(LexisNexisFixtures.example_ddp_authentication_config.to_h) }

  # ...
end
```